.github/workflows/Packages - EdFi.Ods.Populated.Template.TPDM.yml

```diff
@@ -0,0 +1,120 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Packages - EdFi.Ods.Populated.Template.TPDM
+
+on:
+  pull_request:
+    branches: [main]
+    
+  workflow_dispatch:
+    inputs:
```

Comment:
The inputs can be removed from workflow_dispatch:

Reply:
yes . Removed it .